recipes-devtools/deepstream/deepstream-6.0_6.0.0-1.bb

```diff
@@ -47,14 +48,6 @@ do_install() {
         install -m 0644 "$f" ${D}${DEEPSTREAM_PATH}/lib/
     done
     ln -sf libnvds_msgconv.so.1.0.0 ${D}${DEEPSTREAM_PATH}/lib/libnvds_msgconv.so
```

Comment:
There's also a new `libnvds_msgconv_audio` library, and we might need a similar symlink for that one.

Also, I don't see you installing the libraries under `lib/cvcore_libs`.

Reply:
I was not sure if we need it. 
I will update the recipe